pkg/types/nutanix/client.go

```diff
@@ -0,0 +1,34 @@
+package nutanix
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	nutanixClient "github.com/terraform-providers/terraform-provider-nutanix/client"
+	nutanixClientV3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+	"k8s.io/klog"
+)
+
+// CreateNutanixClient creates a Nutanix V3 Client
+func CreateNutanixClient(ctx context.Context, prismCentral, port, username, password string) (*nutanixClientV3.Client, error) {
+	ctx, cancel := context.WithTimeout(ctx, 60*time.Second)
+	defer cancel()
+
+	cred := nutanixClient.Credentials{
+		URL:      fmt.Sprintf("%s:%s", prismCentral, port),
+		Username: username,
+		Password: password,
+		Port:     port,
+		Endpoint: prismCentral,
+		Insecure: false,
```

Comment:
this file appears to be in conflict with what is in https://github.com/openshift/installer/pull/5633/commits/8150c70a911381dfefdf621769b14a8afb1c3c22#diff-aa325f2e2fa65a7eb01b9fb635844901f1d27126471742a88661c666feb83f8aR22 .  Is the intention to use client.go from 5633?

Reply:
Yes, when 5633 is merged, we can resolve this.